testsuite/integration-arquillian/servers/auth-server/services/testsuite-providers/src/main/java/org/keycloak/testsuite/runonserver/RunHelpers.java

```diff
@@ -41,4 +47,25 @@ public FetchOnServer getRunOnServer() {
         };
     }
 
+    public static FetchOnServerWrapper<CredentialModel> fetchCredentials(String username) {
+        return new FetchOnServerWrapper() {
+
+            @Override
+            public FetchOnServer getRunOnServer() {
+                return (FetchOnServer) session -> {
+                    RealmModel realm = session.getContext().getRealm();
+                    UserModel user = session.users().getUserByUsername(username, realm);
+                    List<CredentialModel> storedCredentialsByType = session.userCredentialManager().getStoredCredentialsByType(realm, user, CredentialRepresentation.PASSWORD);
+                    System.out.println(storedCredentialsByType.size());
```

Comment:
`System.out.println`?

Reply:
Copy/paste from whoever added this in the first place. I just moved the code.

Reply:
https://github.com/keycloak/keycloak/pull/6514/files#diff-0b313f9d585a39b2e3286f6fa414924eL356